lib/src/impl/platform_object.dart

```diff
@@ -15,8 +15,8 @@ abstract class PlatformObject {
   Future<int> _handle;
   int _handleValue; // Only for logging. Otherwise use _handle instead.
 
-  PlatformObject([fetchHandle = true]) {
-    if(fetchHandle) {
+  PlatformObject({bool fetchHandle = true}) {
```

Comment:
I think there should be an assert `assert(fetchHandle != null);` until non-nullable becomes available and this project is migrated.

This is a bit of a pitfall, because the default value is only applied if no parameter is passed. 
If `null` is passed explicitly, `fetchHandle` will be `null`.

That applies to all methods with parameters with default values.

Reply:
18c832d7504d8a6e89a403c834093b454e8062f6